rollup.config.js

```diff
@@ -4,14 +4,16 @@ import replace from '@rollup/plugin-replace'
 import { eslint } from 'rollup-plugin-eslint'
 import { terser } from 'rollup-plugin-terser'
 import { version } from './package.json'
+import sourcemaps from 'rollup-plugin-sourcemaps'
 
 export default {
   input: 'src/main.js',
   output: [
     {
       name: 'clevertap',
       file: 'clevertap.js',
-      format: 'umd'
+      format: 'umd',
+      sourcemap: true
```

Comment:
Do we really need input source map? We don't have any other dependency 😕

Reply:
As Aditi said, we were using it to debug via breakpoints in source maps. We figured it wont be an issue at release since it is a dev dependency.